server/service/src/sync/test/test_data/store.rs

```diff
@@ -121,12 +122,58 @@ fn store_2() -> TestSyncPullRecord {
     }
 }
 
+// Commented out since we're not dealing with HIS store mode right now
```

Comment:
Not sure if we want to keep this in here just in case or not, but we're not dealing with his and drug registration, right now... should I remove Drug Registration from store mode and remove the test...?